src/Services/Concerns/MocksRequests.php

```diff
@@ -79,6 +79,18 @@ public function saveSnapshots()
         return $this;
     }
 
+    /**
+     * Clear snapshots file
+     *
+     * @return void
+     */
+    public function clearSnapShots()
```

Comment:
hate to be a pest, but probably clearSnapshots instead of clearSnapShots

Reply:
Don't need now